framework/db/Connection.php

```diff
@@ -373,6 +373,13 @@ class Connection extends Component
      * @see masters
      */
     public $shuffleMasters = true;
+    /**
+     * @var bool whether to enable logging and profiling of database queries. Defaults to true.
+     * You may want to disable this option in a production environment to gain performance
+     * if you do not need the information being logged.
+     * @since 2.0.12
+     */
+    public $enableQueryLog = true;
```

Comment:
Better will be `enableLogging` to be consistent with the fields we used at Yii 1:
https://github.com/yiisoft/yii/blob/master/framework/db/CDbConnection.php#L226
and at `yii2-mongodb`:
https://github.com/yiisoft/yii2-mongodb/blob/master/Connection.php#L138

It is also be good to have separated `enableProfiling`:
https://github.com/yiisoft/yii/blob/master/framework/db/CDbConnection.php#L232